docs/security.rst

```diff
@@ -0,0 +1,5 @@
+.. _kafkarest_security:
+
+Security Overview
+-----------------
+The REST proxy currently supports API calls over https. At this time, ZooKeeper security and Kafka security are not yet supported.
```

Comment:
If you think it makes sense, we could say to check the configuration options (and link to them) for more details.